### ISSUE TITLE
Update Macbook Pro offering in hardware list

### ISSUE THREAD

**Issue by marnovo** [src-d/guide#310]
The current Mac notebook model we offer in the [hardware list, the 13-in non-touchbar MBP](https://github.com/src-d/guide/blob/cdadebdf481ea788c2d32a29491344aee67ad229/general/available_hardware.md#apple-13-inch-macbook-pro---space-gray) is by now already 1.5 years old (!).

Apple [updated their MBP line a few months ago](https://www.apple.com/newsroom/2018/07/apple-updates-macbook-pro-with-faster-performance-and-new-features-for-pros/), _but_ just the touchbar versions. With the recent launch of the new MB Air, it's [becoming clear for months](https://www.digitaltrends.com/computing/macbook-pro-touch-bar-die/) they're [leaving the non-touchbar version to rot & discontinue](https://appleinsider.com/articles/18/10/31/new-macbook-air-threatens-both-macbook-and-macbook-pro-with-function-keys).

The updated touchbar version (w/the same 16GB RAM and 256GB SSD) is just ~10% more expensive than the non-touchbar yet _vastly_ superior: 

- 4-core 8th gen CPU (vs 2-core 7th gen)
  - Multi-core benchmarks ~75% faster!
- 10% faster GPU
- 100% faster SSD, [more reliable](https://www.imore.com/13-inch-macbook-pro-non-touch-bar-gets-ssd-service-program)
- 4 Thunderbolt 3 ports (vs 2 with quirks)
- [Improved, more reliable and silent keyboard](https://www.macrumors.com/2018/07/19/apple-confirms-2018-mbp-keyboard-prevents-debris/)
- Much better cooling, display, security features, etc.

[Generic model comparison article here](https://appleinsider.com/articles/18/07/18/comparing-the-2018-13-inch-macbook-pro-touch-bar-versus-the-macbook-pro-with-function-keys).

In the end keeping the pretty inferior, objectively less reliable and likely to be discontinued non-touchbar version for a small price difference looks like a quite bad deal… even more so when considering the resale value in the future.

**TL;DR: replace the current Mac offering, the MBP 13" 16GB / 256GB non-touchbar, with the newer & better [MBP 13" 16GB / 256GB touchbar](https://www.apple.com/es/shop/buy-mac/macbook-pro/13-pulgadas-gris-espacial-256gb-de-cuatro-n%C3%BAcleos-a-2,3-ghz).**

P.s.:

Also, I'd _strongly encourage_ to set all the keyboard layouts in laptops to US. We're a fully international company for a while already, US English is our working language, so it becomes a mess when people inherit computers and have to adapt to languages with bizarre layouts.

**Comment by creachadair** [comment]
#309 is also related. This makes sense to me.

**Comment by creachadair** [comment]
I wrote the specs for the 15" model by finding a comparable relative spec to the 13" model. If we are going to bump up the specs for the latter, we should probably also do the former. Would you like me to prep a PR with those changes for your consideration?

**Comment by m09** [comment]
> Also, I'd strongly encourage to set all the keyboard layouts in laptops to US. We're a fully international company for a while already, US English is our working language, so it becomes a mess when people inherit computers and have to adapt to languages with bizarre layouts.

Just a note regarding the layouts: I type in azerty and on the laptop I got (which is not this model), there is no way for me to type `<` and `>`. As you can guess, this is rather annoying to do about anything related to programming. It might therefore be safer to make sure that the layout will be usable for the employee before deciding instead of setting hard guidelines.

I do believe that whenever the differences are small it's fine to opt for the most general layout, but still in some cases it hinders productivity for an unclear future benefit.

**Comment by smacker** [comment]
US keyboard just doesn't have that button:
https://upload.wikimedia.org/wikipedia/commons/thumb/b/b9/KB_France.svg/500px-KB_France.svg.png

**Comment by m09** [comment]
> Whoa, how do you do? But how is this (im)possible with a US keyboard?

I use [X11 unicode input](https://en.wikipedia.org/wiki/Unicode_input#In_X11_(Linux_and_Unix_variants)) which means I type `Ctrl+Shift+U 033E` for `>` and `Ctrl+Shift+U 033C` for `<`. Which means I don't do HTML without an external keyboard. Overall it's not that big a deal but still obviously annoying. As to why it is impossible to type, it's because the standard US QWERTY layout has one less key than the standard AZERTY layout (between `Shift` and `z`, we have the key for `>` and `<`.

> This proposal came from a general view a while ago, when a lot of the company had to either relearn how to type with the super quirky/inefficient Spanish layout or just literally ignore the key labels and map to a regular QWERTY layout (my case). So it's about choosing the "least" overall inefficiency/loss in resale value in the long run.

I completely agree that we should aim for the best tradeoff between comfort and resale/reuse value. It's just that we have to take care to correctly weight the comfort depending on the specific layouts involved.

**Comment by dpordomingo** [comment]
I also wonder if inheriting laptops would be a priority for a company that is growing and is mostly remote, and there is no that much rotation, and the expected life of a laptop should be less than the expected for a developer in the company.

**Comment by smola** [comment]
I think we can move on with the Macbook Pro offering update and move the keyboard layout discussion to a different issue? The layout issue affects all laptops as well as external keyboards.
